readme-contents/HealthChecks.md

```diff
@@ -0,0 +1,17 @@
+# Health Checks
+
+## What is provided?
+
+By including the library you get 2 abstract classes to extend to create your own health checks against external APIs.
```

Comment:
Think it would be good to clarify here that these are ping checks, not health checks.
I was wondering if we could do anything automatically here like we do in the spring boot sqs project i.e. it would be great if the health (ping) checks are automatically created for people instead of having to remember to create them?

Reply:
r.e. doing something automatically... Quite possibly, but it ties into automating creationg of web clients and configuring the base uri of external APIs. I think that warrants it's own ticket and needs thinking through.